pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -0,0 +1,161 @@
+package cis
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/rancher/rancher/pkg/app/utils"
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/rancher/rancher/pkg/systemaccount"
+	rcorev1 "github.com/rancher/types/apis/core/v1"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	mgmtv3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	projv3 "github.com/rancher/types/apis/project.cattle.io/v3"
+	"github.com/rancher/types/config"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type cisScanHandler struct {
+	clusterLister                v3.ClusterLister
+	mgmtCtxClusterClient         v3.ClusterInterface
+	mgmtCtxProjClient            mgmtv3.ProjectInterface
+	mgmtCtxAppClient             projv3.AppInterface
```

Comment:
nit: we normally just name them as `apps` for AppInterface and  `appLister` for AppLister

Reply:
I used to do the same, but I liked this new approach used in monitoring: https://github.com/rancher/rancher/blob/master/pkg/controllers/user/monitoring/appHandler.go#L13 and took it to next level :)